### PR TITLE
workflows: debug Gitlab CI trigger

### PR DESCRIPTION
**Pull request by jrusz**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Review by Gundersanne — DISMISSED** [review]

File: .github/workflows/trigger-gitlab.yml [hunk 1]
```diff
@@ -76,6 +76,7 @@ jobs:
 
       - name: Push to gitlab
         run: |
+          set -x
           mkdir -p ~/.ssh
           echo "${SCHUTZBOT_SSH_KEY}" > ~/.ssh/id_rsa
```
Comment (Gundersanne) [inline .github/workflows/trigger-gitlab.yml:81]: Are we sure github will mask the private key here?

Comment (jrusz) [reply .github/workflows/trigger-gitlab.yml:81]: Huh, good question, let my try in my testing repo.

Comment (jrusz) [reply .github/workflows/trigger-gitlab.yml:81]: I defined it as repository secret, that's not going to work?

Comment (Gundersanne) [reply .github/workflows/trigger-gitlab.yml:81]: change 
```
    env:
      SCHUTZBOT_SSH_KEY: I_DONT_CARE
```
to 
```
    env:
      SCHUTZBOT_SSH_KEY: ${{ secrets.SECRET }}
```

Comment (Gundersanne) [reply .github/workflows/trigger-gitlab.yml:81]: cool, seems alright

**Review by Gundersanne — APPROVED** [review]

**Pull request merged**